build_tools/build_detect_platform

```diff
@@ -774,6 +786,7 @@ echo "JEMALLOC_LIB=$JEMALLOC_LIB" >> "$OUTPUT"
 echo "ROCKSDB_MAJOR=$ROCKSDB_MAJOR" >> "$OUTPUT"
 echo "ROCKSDB_MINOR=$ROCKSDB_MINOR" >> "$OUTPUT"
 echo "ROCKSDB_PATCH=$ROCKSDB_PATCH" >> "$OUTPUT"
+echo "ROCKSDB_OPENSSL_AES_CTR=$ROCKSDB_OPENSSL_AES_CTR" >> "$OUTPUT"
```

Comment:
I do not think this line is necessary.  There is nothing set in this file to this variable, and nothing in the Makefile refers to it

Reply:
This sets the environment variable that make passes to the compilers.

Reply:
removed it and rebuild.  seems to all still work.  checking in without line.